### ISSUE TITLE
Will this project be maintained anymore?

### ISSUE THREAD

**Issue by BreakingLead** [yongkyuns/noon#4]

**Comment by yongkyuns** [comment]
Yes—the project is actively maintained. Current architecture/roadmap work is in #952 with execution trackers #953–#956. Closing this old maintenance-status question as answered.
